python/lsst/sims/maf/metricBundles/metricBundleGroup.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, bundleDict, dbObj, outDir='.', resultsDb=None, verbose=True,
         self.bundleDict = bundleDict
         # Check the dbObj.
         if not isinstance(dbObj, db.Database):
-            raise ValueError('dbObj should be an instantiated lsst.sims.maf.db.Database (or child) object.')
+            warnings.warn('Warning: dbObj should be an instantiated lsst.sims.maf.db.Database (or child) object.')
```

Comment:
I wanted to run unit tests without a dbObject. Since we can now pass in simData manually, I think it makes sense.